draft-birkholz-scitt-architecture.md

```diff
@@ -483,10 +483,10 @@ TS implementations MUST indicate their support for registration policies and MUS
 
 Policy Name | Required `reg_info` attributes | Implementation
 ---|---|---
-TimeLimited | `register_by: uint` | Returns true if now () < register_by at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts
+TimeLimited | `register_by: uint` | Returns true if now () < `register_by` at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts. The value provided for `register_by` MUST be a Unix timestamp, i.e., a `uint` representing the number of seconds since 00:00:00 UTC, 1 January 1970, without leap seconds.
 Sequential | `sequence_no: uint` | First, lookup in the ledger for claims with the same issuer and feed. If at least one is found, returns true if and only if the `sequence_no` of the new claim is the highest `sequence_no` in the existing claims incremented by one. Otherwise, returns true if and only if `sequence_no = 0`.
-Temporal | `issuance_ts: uint` | Returns true if and only if there is no claim in the ledger with the same issuer and feed with a greater `issuance_ts`
-NoReplay | None | Returns true if and only if the claim doesn't already appear in the ledger
+Temporal | `issuance_ts: uint` | Returns true if and only if there is no claim in the ledger with the same issuer and feed with a greater `issuance_ts` and now () > `issuance_ts` at registration time. The value provided for `issuance_ts` MUST be a Unix timestamp, i.e., a `uint` representing the number of seconds since 00:00:00 UTC, 1 January 1970, without leap seconds.
```

Comment:
I'm looking closer for the first time -- is there no need for a reg-info to be compact?
(You are using text string labels, which are less compact than integer labels would be, which is OK if they don't have to be compact.)

Reply:
This may be desirable, but I call it out of scope for this PR. I've opened an issue to track it: https://github.com/ietf-scitt/draft-birkholz-scitt-architecture/issues/43